docs/_static/style.css

```diff
@@ -669,6 +673,33 @@ aside .material-icons {
   align-items: baseline;
 }
 
+/* sidebar search box stuff */
+
+.sidebar-search-wrapper {
+  display: flex;
+  align-items: stretch;
```

Comment:
flex layouts won't apply with a fixed `width:` set

Reply:
Should be fixed now, I think.

Reply:
updated slightly in https://github.com/DisnakeDev/disnake/pull/963/commits/3107931fe64bc45a829f2c62e1a9c4f730c4166d as discussed